packages/wonder-blocks-tokens/src/theme/primitive/border.ts

```diff
@@ -24,4 +24,8 @@ export const border = {
         medium: remToPx(sizing.size_020),
         thick: remToPx(sizing.size_040),
     },
+    offset: {
```

Comment:
Do you think the border offset token would be helpful as design tokens too in Figma? If so, this is something we can coordinate with Caitlyn on!

Also, do we want to document the new border offset tokens? Or use it internally only for now? (can be done separately!)

Reply:
Also, since we use it for setting the text underline offset, I'm wondering if the underline offset token should be a token related to `font`, rather than `border`! Borders don't really have offsets, but they could apply to outlines! What do you think?